corehq/apps/app_manager/xform.py

```diff
@@ -646,9 +648,15 @@ def all_media_references(self, lang):
         return images + video + audio + inline_video
 
     def get_instance_ids(self):
+        def _get_instances():
+            return itertools.chain(
+                self.model_node.findall('{f}instance'),
+                self.model_node.findall('instance')
```

Comment:
not sure why this is necessary but it definitely is - could be a bug with the parser?